docs/attributes-registry/gen-ai.md

```diff
@@ -0,0 +1,48 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Large Language Model (LLM)
+
+<!-- toc -->
+
+- [Generic LLM Attributes](#generic-llm-attributes)
+  - [Request Attributes](#request-attributes)
+  - [Response Attributes](#response-attributes)
+  - [Event Attributes](#event-attributes)
+
+<!-- tocstop -->
+
+## Generic LLM Attributes
+
+### Request Attributes
+
+<!-- semconv registry.llm(omit_requirement_level,tag=llm-generic-request) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `gen_ai.llm.request.max_tokens` | int | The maximum number of tokens the LLM generates for a request. | `100` |
+| `gen_ai.llm.request.model` | string | The name of the LLM a request is being made to. | `gpt-4` |
+| `gen_ai.llm.request.temperature` | double | The temperature setting for the LLM request. | `0.0` |
+| `gen_ai.llm.request.top_p` | double | The top_p sampling setting for the LLM request. | `1.0` |
+| `gen_ai.llm.system` | string | The name of the LLM foundation model vendor, if applicable. | `openai` |
```

Comment:
if we keep `gen_ai`, I believe we should have `gen_ai.system` and that'd probably be the only common attribute for the time being